Modules/MUON/MID/src/CalibQcTask.cxx

```diff
@@ -402,12 +402,12 @@ void CalibQcTask::monitorData(o2::framework::ProcessingContext& ctx)
   // auto deads = o2::mid::specs::getData(ctx, "digits", o2::mid::EventType::FET);
   // auto deadrofs = o2::mid::specs::getRofs(ctx, "digits", o2::mid::EventType::FET);
 
-  printf("========================================================== \n");
+  // printf("========================================================== \n");
 
   for (const auto& deadrof : deadrofs) { // loop deadROFs //
-    printf("========================================================== \n");
-    printf("%05d deadROF with first entry %05zu and nentries %02zu , BC %05d, ORB %05d , EventType %02d\n", deadROF, deadrof.firstEntry, deadrof.nEntries, deadrof.interactionRecord.bc, deadrof.interactionRecord.orbit, (int)deadrof.eventType);
-    //  eventType::  Standard = 0, Calib = 1, FET = 2
+    // printf("========================================================== \n");
+    // printf("%05d deadROF with first entry %05zu and nentries %02zu , BC %05d, ORB %05d , EventType %02d\n", deadROF, deadrof.firstEntry, deadrof.nEntries, deadrof.interactionRecord.bc, deadrof.interactionRecord.orbit, (int)deadrof.eventType);
+    //   eventType::  Standard = 0, Calib = 1, FET = 2
```

Comment:
Please consider removing the commented out code for clarity or, since these are logs, using infologger with `Debug` severity and `Trace` level.